Automattic/jetpack
# Sharing: fix all PHPCS errors

Fixes #21970

#### Changes proposed in this Pull Request:

* Fix PHPCS errors in all sharing files.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Start with a site where the Sharing feature is disabled.
* Go to Jetpack > Settings > Sharing and enable the feature.
* Go to Settings > Sharing and ensure that the 2 default buttons are displayed there.
* Enable the sharing buttons on single posts, save your changes, ensure the changes get saved.
* Ensure the buttons are displayed properly on the frontend.
* Ensure you can use the buttons properly.
* Now go back to Settings > Sharing and try to change some settings, add some buttons, add [custom sharing buttons](https://ryanmarkel.com/3004/adding-specific-sharing-services-to-sharedaddy-or-jetpack/). Ensure changes are always saved properly, and reflected on the frontend.
* Try editing some of your posts, and overwriting sharing button status there in the Jetpack sidebar. That should work too.
* Is CI happy?
